altair/api.py

```diff
@@ -232,8 +232,9 @@ def configure_facet_scale(self, *args, **kwargs):
     # Display related methods
     def _ipython_display_(self):
         from IPython.display import display
-        from vega import VegaLite
-        display(VegaLite(self.to_dict()))
+        spec = self.to_dict()
+        data = {'application/vnd.vegalite+json': spec}
```

Comment:
We should add the text/plain one, but because the spec includes the full data, I don't think we should ship it twice...